doc/fundamentals/distributed_computing.md

```diff
@@ -0,0 +1,115 @@
+# Distributed Computing
+
+Distributing computation across multiple workers is a fundamental part of processing any big data workload. There are two main questions to think about when trying to distribute computation:
+
+1) How many workers to spread computation across?
+
+2) How to define which workers are responsible for which data?
+
+## Pachyderm Workers
+
+Before we dive into the above questions, there are a few details you should understand about Pachyderm workers. 
+
+Pachyderm workers are spun up when you create the pipeline and are left running in the cluster waiting for new jobs (data) to be available for processing (committed). This saves having to recreate and schedule the worker pod for every new job.
```

Comment:
Because in the next sentence you say "pod," and this might be the first place in our docs where we are really talking about pods.